plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergV2.java

```diff
@@ -145,20 +145,8 @@ public void testV2TableWithEqualityDelete()
         assertUpdate("CREATE TABLE " + tableName + " AS SELECT * FROM tpch.tiny.nation", 25);
         Table icebergTable = updateTableToV2(tableName);
         writeEqualityDeleteToNationTable(icebergTable);
-        assertQueryFails("SELECT * FROM " + tableName, "Iceberg tables with delete files are not supported: tpch." + tableName);
-    }
-
-    @Test
-    public void testV2TableWithUnreadEqualityDelete()
```

Comment:
why removed?

Reply:
It was there to make sure that delete files only caused a failure if the query actually read files that needed them, but now the query should always work so it's redundant with the test above it